api/nakadi-event-bus-api.yaml

```diff
@@ -1186,15 +1186,17 @@ definitions:
         description: |
           Offset of the event being pointed to. Regarding its format and guarantees:
 
-          - offsets have a fixed size of 26 characters ranging from 0 to F (Hexadecimals).
+          - offsets have a fixed size of 27 characters ranging from 0 to F (Hexadecimals).
```

Comment:
Fixed in https://github.com/zalando/nakadi/pull/403/commits/de05a98ad2fe2fa3323cb2b6e0e39afc00362ff7

I think that after adding an example it became more clear. I would say that the regex could just make it more scary to some people.